src/features/surveys/components/SurveyURLCard.tsx

```diff
@@ -12,15 +12,22 @@ interface SurveyURLCardProps {
 const SurveyURLCard = ({ isOpen, orgId, surveyId }: SurveyURLCardProps) => {
   return (
     <Card style={{ padding: 16 }}>
-      <Box sx={{ display: "flex", justifyContent: "space-between" }}>
+      <Box sx={{ display: 'flex', justifyContent: 'space-between' }}>
         <Typography variant="h5">
           {isOpen ? (
             <Msg id="pages.organizeSurvey.urlCard.open" />
           ) : (
             <Msg id="pages.organizeSurvey.urlCard.preview" />
           )}
         </Typography>
-        <Box sx={{ backgroundColor: isOpen ? "lightgreen" : "lightgray", borderRadius: 5, height: 20, width: 20 }} />
+        <Box
+          sx={{
+            backgroundColor: isOpen ? 'lightgreen' : 'lightgray',
```

Comment:
```
  backgroundColor: isOpen ? Color.success : Color.danger
```
Do we have any default colors that could be used here? I'm thinking some generic value, like the example I gave above `Colors.success` 🤔

Reply:
Great! I found some colors that look similar to the Figma mockups
```
              ? theme.palette.success.main
              : theme.palette.grey['500'],
```
[0b23a8b](https://github.com/zetkin/app.zetkin.org/pull/1054/commits/0b23a8bacad8584575ada3674519d43811f93ef9)